### DISCUSSION TITLE
Can the Search & Compare be decoupled?

### DISCUSSION THREAD

**Discussion by codeforcesmeme** [gitkraken/vscode-gitlens#1203] [Q&A] [answered]
As the title says, How to decouple search and compare into separate tabs?

This was the default behavior pre the moving to source control tab. Now I can't figure how to separate them, and it's getting annoying to click on the small + button then choose whether I will search for a commit or compare heads instead of before where we had a compare tab with the first item which is a big area to click on being create comparison (or something, I don't recall)

**Comment by eamodio** [comment, marked as answer]
They can't be separated, but you can start a compare from the command palette using the _Compare References..._ command (which you can add your own keybinding to)

**Comment by codeforcesmeme** [comment]
> They can't be separated, but you can start a compare from the command palette using the _Compare References..._ command (which you can add your own keybinding to)

Will this ever an option to separate them, it just seems counter-intuitive? I think the design option was forced just to make the gitlens tabs one less tab in the Source Control menu but what if **I** want gitlens the way it was before the move to Source Control with all the tabs separated without clutter

**Comment by eamodio** [comment]
What is the compelling reason to not have them combined?

**Comment by euan-gwd** [comment]
not sure if this is relevant here, however, I found it easier to use to have the gitlens in it's own sidebar item.  less clutter in the source control as such i have not updated it.
